lightningd/peer_htlcs.c

```diff
@@ -1986,14 +1986,22 @@ static void commitment_revocation_hook_serialize(
 }
 
 static void
-commitment_revocation_hook_cb(struct commitment_revocation_payload *p,
-			      const char *buffer, const jsmntok_t *toks){
+commitment_revocation_hook_cb(struct commitment_revocation_payload *p STEALS){
 	wallet_penalty_base_delete(p->wallet, p->channel_id, p->commitnum);
 }
 
-REGISTER_PLUGIN_HOOK(commitment_revocation, PLUGIN_HOOK_CHAIN,
+static bool
+commitment_revocation_hook_deserialize(struct commitment_revocation_payload *p,
+				       const char *buffer,
+				       const jsmntok_t *toks)
+{
+	return true;
+}
+
+
+REGISTER_PLUGIN_HOOK(commitment_revocation,
+		     commitment_revocation_hook_deserialize,
 		     commitment_revocation_hook_cb,
-		     struct commitment_revocation_payload *,
 		     commitment_revocation_hook_serialize,
 		     struct commitment_revocation_payload *);
```

Comment:
This looks like a fixup for the previous commit, not part of ` pytest: Add a test for the commitment_revocation hook ` ?